openlibrary/templates/books/edit.html

```diff
@@ -34,6 +34,7 @@
             note = ""
     <h2 class="editFormTitle">$:title</h2>
     $if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
+        <button type="submit" class="hidden" name="_save" form="addWork" tabindex="-1"></button>
```

Comment:
Yep; can't find a better solution. Because the consequences of this are a little extreme, let's see if we can avoid this happening in the future!

Could you (1) create a `$def` like `render_submit_button(hidden=False)`, so that the two submit buttons always have the correct values?

And (2) try placing the hidden button as the first element in the `addWork` `<form>`. That should hopefully prevent this issue from ever happening again!

Reply:
Oh, and (3) add a comment about why it's necessary; so some future person doesn't "helpfully" delete the "unused hidden button"